src/uct/ib/dc/base/dc_iface.h

```diff
@@ -47,8 +47,9 @@ typedef struct uct_dc_dci {
 
 
 typedef struct uct_dc_fc_sender_data {
-    uintptr_t                     ep;
+    uint64_t                      ep;
     union ibv_gid                 gid;
+    int                           is_global;
```

Comment:
since this is a wire format, better to put "is_global" before "gid" because it indicates whether the gid is valid, and make it uint8_t. also, declare the whole structure as UCS_S_PACKED.

Reply:
done